Examples/Image/Detection/FastRCNN/BrainScript/A2_RunWithPyModel_distributed.py

```diff
@@ -0,0 +1,220 @@
+﻿# Copyright (c) Microsoft. All rights reserved.
+
+# Licensed under the MIT license. See LICENSE.md file in the project root
+# for full license information.
+# ==============================================================================
+
+from __future__ import print_function
```

Comment:
Is this file copied from non-distributed version? Code copy would increase maintenance cost in future, please either modify the non-distributed version to support parallel training, or import functions from it to reduce code duplication.

Reply:
As you see in [this script](https://github.com/Microsoft/CNTK/tree/master/Examples/Image/Classification/ConvNet/Python), there are similar scripts for usual learning and distributed one. As a policy of CNTK repository, should we merge this script to one?